.travis/is-bump-commit.sh

```diff
@@ -2,7 +2,7 @@
 
 isBumpCommit() {
   msg=`git log --pretty=format:'%s' -1`
-  if [ "$msg" = "Automated version bump" ]
+  if [[ "$msg" =~ ^\[pr-bumper\]* ]]
```

Comment:
Nope, not unless you did
```
git commit -m "[pr-bumper] fix config"
```
It's checking that the msg starts with `[pr-bumper]` I included the brackets for just that reason.